File: API/Helper/Params/Transaction/PickingScanParam.cs
namespace API.Helpers.Params.Transaction
{
    public class PickingSCanBaseParam
    {
        public string Manuf { get; set; }
        public string Iono { get; set; }
        public string ManNo { get; set; }
        public string PurNo { get; set; }
        public string Size { get; set; }
    }

    public class PickingScanParam : PickingSCanBaseParam
    {
        public string DeclarationNo { get; set; }
        public string Invno { get; set; }
        public string StartReleaseDate { get; set; }
        public string EndReleaseDate { get; set; }

        public string CurrentUser { get; set; }
    }

    public class PickingScanItemParam
    {
        public string Iono { get; set; }
        public string ManNo { get; set; }
        public string PurNo { get; set;}
        public string Size { get; set; }
    }
}

File: SPA/src/app/_core/_helpers/params/transaction/pickingScanParam.ts
export interface PickingSCanBaseParam {
    manuf: string;
    iono: string;
    manNo: string;
    purNo: string;
    size: string;
}

export interface PickingScanParam extends PickingSCanBaseParam {
    declarationNo: string;
    invno: string;
    releaseDate: string;
    startReleaseDate: string;
    endReleaseDate: string;

    isEditOrDetail?: boolean; // Nếu là True: Edit , False: Detail
}

export interface PickingScanSource {
    currentPage: number;
    param: PickingScanParam,
    source: PickingScanParam
}
export interface PickingDetailParam {
    iono: string;
    releaseDate: string;
    startReleaseDate: string;
    endReleaseDate:string;
    storageOutDate: string;
    size: string;
    manno: string;
    purno: string;
}